server/spec/test_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'spec_helper'
+require 'candlepin_scenarios'
+require 'unpack'
+
+describe 'Certification verification' do
+  include CandlepinMethods
+  include CertificateMethods
+  include Unpack
+
+
+  it 'certificates should contain all content' do
```

Comment:
how does this file relate to the change? was it added by accident?

Reply:
It doesn't, but it keeps sneaking in there when I do a "git add server/spec" without double checking what got added.

Removed.